commands/help.rb

```diff
@@ -7,18 +7,18 @@ def self.help(command)
     when 'build'
       puts <<~EOT
         Build package(s).
-        Usage: crew build [-k|--keep] [-v|--verbose] <package1> [<package2> ...]
+        Usage: crew build [-f|--force] [-k|--keep] [-v|--verbose] <package1> [<package2> ...]
         Build package(s) from source and place the archive and checksum in the `CREW_LOCAL_BUILD_DIR/release/<arch>` directory.
         Build package(s) from source and place the archive and checksum in the current working directory.
+        If `-f` or `--force` is present, interactive prompts will be disabled.
         If `-k` or `--keep` is present, the `CREW_BREW_DIR` (#{CREW_BREW_DIR}) directory will remain.
         If `-v` or `--verbose` is present, extra information will be displayed.
       EOT
     when 'check'
       puts <<~EOT
-        Check package(s) for syntax errors, and copy local packages to the chromebrew directory.
+        Check package(s) for syntax errors.
```

Comment:
it still copies local packages to the chromebrew directory.

Reply:
explaining the details is kinda the point of the help command...

Reply:
Not necessary to explain.

Reply:
The reason I say this is the main purpose of the command is to check for syntax errors.  If the other thing is mentioned, it clouds up the clarity.

Reply:
I disagree, but I'm not going to force the matter.